mathjaxandroid/src/main/java/de/timfreiheit/mathjax/android/MathJaxView.java

```diff
@@ -119,12 +129,16 @@ public void onPageFinished(WebView view, String url) {
      * called when webView is ready with rendering LaTex
      */
     protected void rendered() {
-        handler.postDelayed(new Runnable() {
-            @Override
-            public void run() {
-                mWebView.setVisibility(View.VISIBLE);
-            }
-        }, 100);
+        if (onMathJaxRenderListener != null) {
```

Comment:
- the webview behavior should be independent from listeners
- the listener should be triggered within the handler callback to make sure it is called on the main thread